website/docs/pubsub.md

```diff
@@ -0,0 +1,188 @@
+---
+id: pubsub
+title: Pubsub
+---
+
+`Feature State`: Gatekeeper version v3.13+ (alpha)
+
+> ❗ This feature is still in alpha stage, so the final form can still change (feedback is welcome!).
+
+## Motivation
+
+Prior to this feature, there were two ways to get audit violations. One is to look at constraints status and the other is to look at audit pod logs and get the logged audit violations. Both of these approach have limitations as described below.
+
+Limitations of getting audit violations from constraint status:
+
+- To reduce in memory consumption of audit and avoid violating etcd limit of 1.5MB per resource, gatekeeper only reports up-to 500 violations on constraint template. In other words, one might not get all the violations from looking at constraint.
```

Comment:
```suggestion
- To reduce in-memory consumption of Gatekeeper audit pod and to avoid hitting the etcd limit of 1.5MB per resource, gatekeeper recommends configuring a limit up-to 500 violations on constraint template. As a result, it’s possible that not all the violations are reported as part of the constraint status.
```